src/android/LocalNotification.java

```diff
@@ -306,20 +305,23 @@ private void actions (JSONArray args, CallbackContext command) {
      * @param toasts  The notifications to schedule.
      * @param command The callback context used when calling back into
      *                JavaScript.
+     * @param requestExactAlarmPermission whether should request exact alarm permission or not
+     *                                    (e.g. send false if already checked prior)
      */
-    private void schedule (JSONArray toasts, CallbackContext command) {
+    private void schedule (JSONArray toasts, CallbackContext command, boolean requestExactAlarmPermission) {
```

Comment:
is `requestExactAlarmPermission` necessary? it seems it's always passed false

Reply:
true is passed in this line: https://github.com/OutSystems/cordova-plugin-local-notifications/pull/35/files#diff-c726fe0b5dd866e02057ca83ce940d4ea190d99b7bc7b0db2ca0fd8d5b6cba28R173

Basically should only request the permission if it's executing the command, otherwise if the app is returning to foreground from the alarm permission request or the notifications permission request, should not.